pypeit/pypeit.py

```diff
@@ -556,16 +536,38 @@ def extract_one(self, frames, det, bg_frames=[], std_outfile=None):
         # Object finding, first pass on frame without sky subtraction
         self.maskslits = self.caliBrate.maskslits.copy()
 
-        self.redux = reduce.instantiate_me(self.spectrograph, self.caliBrate.tslits_dict, self.mask, self.par,
+        self.redux = reduce.instantiate_me(self.spectrograph, self.caliBrate.tslits_dict,
+                                           self.mask, self.par,
                                            ir_redux = self.ir_redux,
                                            objtype=self.objtype, setup=self.setup,
                                            det=det, binning=self.binning)
 
+        # Prep for manual extraction
```

Comment:
So I think this scifile parsing that you are doing below should be a method in the fits table, since and not stored in th parset in this way. That is because it is stuff tied to an individual frame in the .pypeit file and as such is should be treated in exactly the same way as the comb_id, and calib_id. In other words, there should be a fitstable.get_manual(frame) or something. Please consult with @kbwestfall. What with parsing the sci_file etc. and stuffing this into the parset is at odds with how we treat the other meta-information that the user puts in the pypeit file.

Reply:
done.